corehq/apps/data_cleaning/templates/data_cleaning/modals/confirm_apply.html

```diff
@@ -10,7 +10,7 @@
   hx-post="{{ request.path_info }}{% querystring %}"
   hq-hx-action="apply_all_changes"
   hx-target="{% if table.container_id %}#{{ table.container_id }}{% else %}.table-container{% endif %}"
-  hx-swap="outerHTML"
+  hx-swap="none"
```

Comment:
Just curious how does this compare to not specifying hx-swap at all?

Reply:
if you don't specify hx-swap it defaults to innerHTML which is...definitely not what we want 😅